sys/zuse.hoon

```diff
@@ -7942,4 +7936,148 @@
       --
     --
   --
+:: |dawn: pre-boot request/response de/serialization and validation
+::
+++  dawn
+  =>  |%
+      :: +live: public network state of a ship
+      ::
+      +$  live  (unit [=life breach=?])
+      :: +seed: private boot parameters
+      ::
+      +$  seed  [who=ship lyf=life key=ring sig=(unit oath:pki:jael)]
+      --
+  |%
+  :: |give:dawn: produce requests for pre-boot validation
+  ::
+  ++  give
```

Comment:
Maybe `=, ethereum, html, mimes:html` in this?

Reply:
I'm torn when it comes to adding lots of `=,`. I also think we need a review of the boundaries between the different ethereum-related cores (`|ethereum` vs `|ethe`) in particular).

Reply:
I generally try to =, any namespace I enter more than like twice, but a rule for this should definitely be part of the style standard.

Strongly agreed about `ethereum` and `ethe`. I split it up like that because types vs functions, which seemed to be a zuse convention, but it *is* a bit unwieldy as-is.